packages/main/src/MultiComboBox.ts

```diff
@@ -742,6 +742,12 @@ class MultiComboBox extends UI5Element implements IFormInputElement {
 		}
 	}
 
+	_onPopoverFocusOut() {
```

Comment:
I think we can go without this method and set this._tokenizer.expanded = false in the _afterClose method. We already have a check and collapse the tokenizer for phone devices, but we can do it for desktop as well.

Reply:
As discussed the method will stay.